ansible/roles/internal_lb/tasks/main.yml

```diff
@@ -1,12 +1,27 @@
 - name: Install pip
-  apt:
-    name:
-      - python3-setuptools
-      - virtualenv
-      - python3-pip
-    update_cache: yes
-  async: 600
-  poll: 30
+  block:
+    - name: Install python tools
+      apt:
+        name:
+          - python3-setuptools
+          - virtualenv
+          - python3-pip
+        update_cache: yes
+      async: 120
```

Comment:
Have we tested how long this usually takes? is 2 minutes enough? And if that's the case I don't think we need to run this as an async task because that change was made since the task took longer than 5 minutes (Might remembered that number wrong though)

Reply:
I would say that it normally takes less than 1 minute, but if it hangs then it takes as long as the timeout. With the old value of 600 I noticed that 2 nodes finished pretty quickly and then the last one timed out when I run into this issue.
We could probably remove the async/poll completely. I just wanted to make as few modifications as possible. :slightly_smiling_face: 
Do you think it would be better to remove it?

Reply:
That's probably a good idea, lets try to merge it like this and see if the solution actually does the trick. And then we can optimize it

Reply:
Cool, keeping it then :slightly_smiling_face: